packages/cactus-plugin-ledger-connector-daml/src/main/typescript/plugin-ledger-connector-daml.ts

```diff
@@ -0,0 +1,301 @@
+        logLevel: this.options.logLevel,
+        connector: this,
+      };
+      const endpoint = new GetPartiesInvolvedEndpoint(opts);
+      endpoints.push(endpoint);
+    }
+
+    this.log.info(`Instantiated endpoints of ${pkgName}`);
+    return endpoints;
+  }
+
+  public async shutdown(): Promise<void> {
+    return;
+  }
+
+  public getOpenApiSpec(): unknown {
+    return OAS;
+  }
+
+  public getInstanceId(): string {
+    return this.instanceId;
+  }
+
+  public async getConsensusAlgorithmFamily(): Promise<ConsensusAlgorithmFamily> {
+    return ConsensusAlgorithmFamily.Authority;
+  }
+
+  public async onPluginInit(): Promise<unknown> {
+    return;
+  }
+
+  public async createContract(req: CreateIOUEndpointRequest): Promise<any> {
+    const createIouUrl = `${this.damlApiUrl}/create`;
+    const participantToken = req.participantToken;
+
+    const responseForIOU = await fetch(createIouUrl, {
+      method: "POST",
+      body: JSON.stringify(req),
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${participantToken}`,
+      },
+    });
+    const responseForIOUResult = await responseForIOU.json();
+
+    return responseForIOUResult;
+  }
+
+  public async exerciseContract(req: ExerciseIOUEndpointRequest): Promise<any> {
+    const exerciseIouUrl = `${this.damlApiUrl}/exercise`;
+    const getRequestPayload = JSON.stringify(req);
+    const parsePayload = JSON.parse(getRequestPayload);
+    const participantHash = parsePayload.participantToken;
+    const requestBody = {
+      templateId: parsePayload.templateId,
+      contractId: parsePayload.contractId,
+      choice: parsePayload.choice,
+      argument: parsePayload.argument,
+    };
+
+    const responseForIOU = await fetch(exerciseIouUrl, {
+      method: "POST",
+      body: JSON.stringify(requestBody),
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${participantHash}`,
+      },
+    });
+    const responseForIOUResult = await responseForIOU.json();
+    return responseForIOUResult;
+  }
+
+  public async queryContract(req: QueryIOUEndpointRequest): Promise<any> {
+    const queryIOUUrl = `${this.damlApiUrl}/query`;
+
+    const getRequestPayload = JSON.stringify(req);
+    const parsePayload = JSON.parse(getRequestPayload);
+    const participantToken = parsePayload.participantToken;
+    const requestBody = {
+      templateIds: parsePayload.templateIds,
+      query: parsePayload.query,
+      readers: parsePayload.readers,
+    };
+
+    const responseForIOU = await fetch(queryIOUUrl, {
+      method: "POST",
+      body: JSON.stringify(requestBody),
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${participantToken}`,
+      },
+    });
+
+    const responseForIOUResult = await responseForIOU.json();
+    return responseForIOUResult;
+  }
+
+  public async queryRawContract(req: QueryRawIOUEndpointRequest): Promise<any> {
```

Comment:
@raynatopedrajeta Please refactor the return type from `any` to the response type of this endpoint in the openapi.json file. This applies to all of the endpoints not just this one.